### DISCUSSION TITLE
Hi everyone

### DISCUSSION THREAD

**Discussion by basimslam** [OpenMined/courses#68] [Introduce Yourself]
My name is Basim and I just joined college for computer science and engineering. I am also doing a parallel degree in data science. I was really impressed by the aim of this organization and I joined this course to be part of this community. Although I don't know much about these right now, I will be trying my best to learn as much as I can. I am hoping to contribute as much as I can with the knowledge I have :) And I think this would be the best place to start with open source contributions. So I can work while studying giving me more idea about how these works. Really excited to be here. I love the passion and enthusiasm the team members are showing. They are very inspiring. And once again, great to meet you all <3

**Comment by iamtrask** [comment]
Hi Basim! Welcome - and I hope to see you around the course - during student years is a great time to go exploring into new areas. Glad you're exploring around here :)

**Comment by yemikifouly** [comment]
Hello Basim! 🙂 Welcome! 

You definitely don't need to know very much to join us. All that is really needed is enthusiasm and curiosity and you definitely have those. I am sure that with those and your areas of study you will contribute a lot to our community. 🙂 As a starting point, as you are going through the course, consider checking out the board often and posting on other students questions if you feel you can help. That would be a great way to deepen you own knowledge and start contributing. 🙂 

Once, again, welcome! It's great to have you here!
